### ISSUE TITLE
[HELP] How to add another data in API result

### ISSUE THREAD

**Issue by nyel-dev** [crocodic-studio/crudbooster#1031]
I'm not sure if this is a CB thing or a pure Laravel thing however, I have customized an API result and did something like this
`
$date_times = DB::table('voyager_pricebook')
`
`->select('pricebook_promo_id', 'pricebook_item','pricebook_deposit_list', 'pricebook_fullprice_list')`
`->where('pricebook_product_id', '=', $postdata["pricebook_product_id"])`
`->orderBy('id','DESC')`
`->get();`

and I am outputting it like this:
`
$result["test"] = $date_times;
`

it returns like this now

![image](https://user-images.githubusercontent.com/13266552/35443655-d334762c-02e6-11e8-85d0-b234c0adf8be.png)

how do I make it add another data inside that object?
I would like it something like this

`
 "test": [`
       ` {`
           ` "pricebook_promo_id": "2",`
           ` "pricebook_item": "Test",`
           ` "pricebook_deposit_list": "0",`
           ` "pricebook_fullprice_list": "0",`
           `"pricebook_test":"test"`
       ` },`
`

basically I would like to add something like this from the result set
`"pricebook_test":"test"`
